Tools/WinMLDashboard/src/view/run/View.tsx

```diff
@@ -94,6 +97,12 @@ class RunView extends React.Component<IComponentProperties, IComponentState> {
     }
 
     private getView = () => {
+        const osInfo = require('os').release()
+        log.info(osInfo);
+        if(osInfo < '10.0.17669') {
+            const message = 'This functionality can only be used for R5-later operating system'
```

Comment:
"This functionality is available on Windows 10 October 2018 Update (1809) or newer version of OS."